__tests__/app.test.js

```diff
@@ -91,3 +90,38 @@ describe("app", () => {
     });
   });
 });
+
+describe("app", () => {
+  describe("/api", () => {
+    describe("GET: /api/users", () => {
+      test("200: should return an array of objects containing users that match test data", () => {
+        return request(app)
+          .get("/api/users")
+          .expect(200)
+          .then((response) => {
+            expect(response.body.length).toBe(4);
+            response.body.forEach((user) => {
+              expect(Array.isArray(user)).toBe(false);
+              expect(typeof user).toBe("object");
+              expect(user).toEqual({
+                username: expect.any(String),
+                name: expect.any(String),
+                avatar_url: expect.any(String),
+              });
+            });
+          });
+      });
+      test("404 if there is a reference error or typo, returns error message", () => {
```

Comment:
I like that you're thinking of error handling good job.
But isn't this already tested and covered in a previous test?